lib/src/main/java/io/ably/lib/http/AsyncHttp.java

```diff
@@ -33,6 +32,19 @@
 	}
 
 	/**
+	 * Async HTTP PUT for Ably host, with fallbacks
+	 * @param path
```

Comment:
I am not sure how Javadocs work but is there any value in these arguments being defined with no comments or types?  Is this not just additional noise in the code if it adds nothing for the end-user?